### PR TITLE
Rename Hemi testnet tokens

### PR DESCRIPTION
**Pull request by leanazulyoro**
Updates Hemi testnet tokens according to the doc https://docs.google.com/document/d/1xXTnfKhK_s6yvjOhPUZRfGo4qiqYsc5qH1u0Vv56q0U/edit  (basically thDAI->DAI and thETH->ETH)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hemilabs/smart-order-router",
-  "version": "3.26.1",
+  "version": "3.26.2",
```
Comment (gndelia) [inline package.json]: I think we were using `beta-*` versions to our forked packages for the time being. See [universal-router-sdk](https://github.com/hemilabs/universal-router-sdk/blob/0a0b4fc0de85a833c2e0a2f926edc11e5b4ae6cd/package.json#L3). However, I do see we did not that for `sdk-core` 🤔

Comment (leanazulyoro) [reply package.json]: Changed to 3.26.1-beta.2
I know it did not say 3.26.1-beta.1 in the package.json before, but that is the published version that's used in routing-api:
Captura de pantalla 2024-05-08 a la(s) 10 19 24

Comment (gndelia) [reply package.json]: ok, we may have updated it in a branch published but never merged it to master 🤦🏽

**Review by gndelia — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]
LGTM

**Pull request merged**